src/Styleguide/Components/Lightbox/Lightbox.tsx

```diff
@@ -0,0 +1,286 @@
+import { color, space } from "@artsy/palette"
+import FadeTransition from "Components/Animation/FadeTransition"
+import { bind, once, throttle } from "lodash"
+import React from "react"
+import ReactDOM from "react-dom"
+import styled from "styled-components"
+import { Box } from "Styleguide/Elements/Box"
+import { Flex } from "Styleguide/Elements/Flex"
+import { CloseButton } from "./CloseButton"
+import { Slider, SliderProps } from "./LightboxSlider"
+
+const KEYBOARD_EVENT = "keyup"
+const ZOOM_PER_CLICK = 1.4
+const HIDE_ZOOM_SLIDER_AFTER = 2500
+
+const DeepZoomContainer = styled.div`
+  position: fixed !important;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  background-color: ${color("black100")};
+`
+
+export interface LightboxProps {
+  deepZoom: {
+    Image: {
+      xmlns: string
+      Url: string
+      Format: string
+      Overlap: number
+      TileSize: number
+      Size: {
+        Width: number
+        Height: number
+      }
+    }
+  }
+}
+
+export interface LightboxState {
+  shown: boolean
+  element: Element
+  viewer: any
+  deepZoomRef: any
```

Comment:
Looks like the type is `RefObject<…>` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/91f8a6b7d1ef61cc543d0901dcc004b846283345/types/react/index.d.ts#L558